setup.py

```diff
@@ -42,7 +47,6 @@
         'validictory >= 0.8.0, != 0.9.2',         # error messages
         'decorator >= 3.3.1',                     # > 3.0 likely work, but not on pypi
         'mutagen >= 1.18',                        # EasyID3 module renaming
-        'protobuf >= 2.4.1',                      # 2.3.0 uses ez_setup?
         'requests >= 1.1.0, != 1.2.0, != 2.2.1, < 2.8.0',  # session.close, memory view TypeError
         'python-dateutil >= 1.3, != 2.0',         # 2.0 is python3-only
         'proboscis >= 1.2.5.1',                   # runs_after
```

Comment:
I cut gpsoauth 0.1.0 with the compatibility changes (github doesn't let me comment on that line).